pkg/kubelet/cm/dra/manager.go

```diff
@@ -126,26 +116,17 @@ func (m *ManagerImpl) PrepareResources(pod *v1.Pod) error {
 
 				klog.V(3).InfoS("NodePrepareResource succeeded", "response", response)
 
-				// NOTE: Passing CDI device names as annotations is a temporary solution
-				// It will be removed after all runtimes are updated
-				// to get CDI device names from the ContainerConfig.CDIDevices field
```

Comment:
I don't see this comment anymore. I understand that generateCDIAnnotations call was moved to another file, but the comment was either forgotten or dropped in purpose. Can you elaborate?

Reply:
mmm, I forgot it in the rebase. I will fix it in the next PR spin.